### PR TITLE
Revision edits

### LINKED ISSUE #231: Revision edits checklist
- [x] Please limit the revised manuscript to no more than 6000 words, excluding acknowledgments, references and figure legends, which should be shortened as much as possible.
- [x] Figures: You are allowed up to 6 main-text figures or tables. Figure and table legends should be concise. We discourage the use of red/green for contrast. Please follow our figure guidelines, [found here](https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.nature.com%2Fdocuments%2FNRJs-guide-to-preparing-final-artwork.pdf&data=05%7C01%7Ccasey.s.greene%40cuanschutz.edu%7Cc08d879944fb4cfe4f3108db09879c60%7C563337caa517421aaae01aa5b414fd7f%7C0%7C0%7C638114250677626824%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=l9zUM4OzjETILZhYtNsynPspGhyHFcUMb9iPWPjhw0A%3D&reserved=0). Please upload figures individually, with editable layers intact. AI, EPS, PDF files preferred.
- [x] limit the number of references to 100
- [x] identify those references (≤ 10% of the total) that, in your opinion, represent key contributions to the field. Please add a short note (1–2 sentences) at the end of each of these references, explaining their importance. Only papers that have been published or accepted by a named publication or recognized preprint server should be in the numbered list.
- [x] Check for citation of unpublished work. Unpublished meeting abstracts, personal communications and manuscripts under consideration (and not formally accepted) may be cited only in the main text.
- [x] If you are including any Supplementary Information, please complete and return [this inventory document](https://nam02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.nature.com%2Fdocuments%2FInventory_of_Supporting_Information_2021.docx&data=05%7C01%7Ccasey.s.greene%40cuanschutz.edu%7Cc08d879944fb4cfe4f3108db09879c60%7C563337caa517421aaae01aa5b414fd7f%7C0%7C0%7C638114250677626824%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=T3cInTNMoYiiq8q0YSY7rZoriA1L9YG3SDJL4HZ6JZA%3D&reserved=0) with your final submission files. Instructions for organizing the Supplementary Information are also included in this document.
- [x] Competing Interests:
In the interests of transparency and to help readers to detect potential bias, Nature and the Nature monthly research journals now require all authors to declare any potential competing financial interests in relation to papers accepted for publication. This statement will be published after the acknowledgments section of your article. Please ensure that the information in the "Competing Financial Interests" section of the manuscript data online is correct and that any competing interests described there are also listed in a paragraph at the end of your manuscript. This will appear as a web supplement. More detailed information about this policy and the form can be found on the web at [http://www.nature.com/authors/editorial_policies/competing.html](https://nam02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.nature.com%2Fauthors%2Feditorial_policies%2Fcompeting.html%253C%2Fa%253E&data=05%7C01%7Ccasey.s.greene%40cuanschutz.edu%7Cc08d879944fb4cfe4f3108db09879c60%7C563337caa517421aaae01aa5b414fd7f%7C0%7C0%7C638114250677783063%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ae7VsjLLCdv3ZjT87GyCxrBFz%2FNdkDabghtr2lwllYs%3D&reserved=0) and in a 2006 editorial [http://www.nature.com/nmeth/journal/v3/n11/full/nmeth1106-869.html](https://nam02.safelinks.protection.outlook.com/?url=http%3A%2F%2Fwww.nature.com%2Fnmeth%2Fjournal%2Fv3%2Fn11%2Ffull%2Fnmeth1106-869.html%253C%2Fa%253E.&data=05%7C01%7Ccasey.s.greene%40cuanschutz.edu%7Cc08d879944fb4cfe4f3108db09879c60%7C563337caa517421aaae01aa5b414fd7f%7C0%7C0%7C638114250677783063%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=XbtKcXtaf%2F4hFpEUhpi3qpaJfH%2F5yUIaSwDhriuO9tI%3D&reserved=0).
- [x] consult with your coauthors to ensure that all names, affiliations, and titles are represented correctly
- [x] Acknowledgements and Contributions:
Acknowledgements :
• Refer to individuals by first initial and last name, without titles or affiliations
• If any listed authors received grants or funding then that information must be provided. Please list each grant number, followed by the initials of the author who received it, in the Acknowledgements section of your paper.
Contributions: If there is more than one author:
• Please keep your Author Contributions statement as concise as possible
• Specify the individual contributions of each coauthor.
Example: "A.P.M. contributed Y and Z; B.T.R. contributed Y and W,” etc.
• Please ensure that author initials listed match the author list
• Distinguish authors with the same initials in some way
Example: “J. Doe contributed Y and Z; J. Dee contributed Y and W,” etc.
- [x] Manuscript Order
All manuscripts must include the following sections, ordered as listed:
• Title
• Author List and affiliations (including present addresses, equal contributions statement, and corresponding author email(s))
• Abstract (150 words, no references)
• Main Text (with subheadings)
• Data and Code Availability Statement (if applicable)
• Acknowledgements
• Author Contributions (if there is more than one author)
• Ethics Declaration
• References (for main text only)
• Figure legends (for main text only)
• Tables (note: tables should be pasted into Word files as editable tables, not as images)
• Boxes

### PR DESCRIPTION
**Pull request by jaybee84**
### Purpose

Resolves #231 , #232 , #232 

### Directions for reviewers

HI All, I have ported all recent edits to GH from the word doc. Please review all changes and check if they address reviewer comments. No need to edit the word doc anymore. I was able to build the docx version locally and will be happy to do the track changes once we have finalized all edits.

#### Which areas should receive a particularly close look?

* All new edits
* Please make sure to check the Author contribution section to make sure everything looks good. 

#### Is the pull request ready for review?

YES

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [ ] All terms that need to be added to glossary have been added to #50 
- [ ] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

**Comment by jaybee84** [comment]
@jaclyn-taroni please do your review/edits on this PR

File: content/16. box_3_MLtasks.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+### (c) Clinical trial design improvement: 
+Optimized study design and identification of appropriate trial participants can greatly reduce costs while increasing the probability of successful outcomes for clinical trials. ML approaches can benefit clinical trial study design. For example, unsupervised ML approaches can be used to identify sub-groups of patients who are more likely to respond well to a particular treatment. Supervised ML approaches can be used to predict drug response in rare disease patients.
+
+### (d) Molecular subtyping of disease: 
+Rare diseases can be complicated and often show overlapping and heterogenous phenotypes. ML approaches can be used to identify molecular subtypes of the disease for better understanding. For example, unsupervised ML approaches can help identify new subtypes of a rare disease using molecular and genetic data. The same approaches can help identify the important molecular features that define the subtypes. 
+
+### (e) Patient prognosis prediction:
```
Comment (jaclyn-taroni) [inline content/16. box_3_MLtasks.md]: I don't believe this is currently referenced anywhere (source: searching all documents for "Box 3").

**Review by jaclyn-taroni — COMMENTED** [review]
Returning a comment I had here. I also wanted to ask – @jaybee84 would you be able to summarize how you picked what went into Box 3? I know we discussed using what is in the manuscript to guide what tasks to include. What I'm trying to understand: are the tasks that got picked and where they are referenced in the text are related to _citations of specific studies_? I.e., if you're referencing diagnostic decision support in the main text – "(Box 3a"), is that because one of the studies cited in that passage fit that bill? Asking to focus my review/edits 😸

**Comment by jaybee84** [comment]
> I.e., if you're referencing diagnostic decision support in the main text – "(Box 3a"), is that because one of the studies cited in that passage fit that bill?

@jaclyn-taroni This is exactly the case.

File: content/16. box_3_MLtasks.md (hunk 1 above)
Comment (jaybee84) [reply content/16. box_3_MLtasks.md]: This is true. We have not touched upon Box 3e in our cited studies.

**Comment by jaybee84** [comment]
@jaclyn-taroni I realize now that I responded to only half your comment above:

The main way I arrived at the content of the box was by reviewing the main citations to see what were the major tasks those papers were referring to. Box 3a-d are cited in the sections that had references to papers which intended to do those tasks. Box 3e is the only one which is not referenced in our sections, but I added this due to my prior knowledge about a recent paper in neurofibromatosis which is partially focused on this problem.

**Comment by jaclyn-taroni** [comment]
👍🏻  keeping this in mind as I review where the references to Box 3 are inserted and how the tasks are defined (in progress)

**Comment by jaybee84** [comment]
@jaclyn-taroni Thanks! If you get the chance to cut some words, that would be great too. We are currently over the word limit. I am also planning to do word count edit once your PR is in.

**Comment by jaybee84** [comment]
@jaclyn-taroni @allaway our one week extension request was accepted.

@jaclyn-taroni I am hoping to send the updated manuscript to Casey and Justin by Feb 22 (wed) so that they have one week to review before our submission deadline is up. Please let me know if this timeline sounds ok, or if I can help in any way to keep us on track with the timeline :)

**Comment by jaclyn-taroni** [comment]
@jaybee84 Do you want to update the LASSO reference I added as a TODO in my branch here? That also extends to anything else that was edited in the Word doc but didn't get added here that I missed.

I am aiming to wrap up my edits today and potentially address #234 (but I think we can all work on #234 in parallel).

**Comment by jaybee84** [comment]
@jaclyn-taroni 
> Do you want to update the LASSO reference I added as a TODO in my branch here?  

-added

>That also extends to anything else that was edited in the Word doc but didn't get added here that I missed.

WIP

**Comment by jaybee84** [comment]
> That also extends to anything else that was edited in the Word doc but didn't get added here that I missed.

@jaclyn-taroni  between your edits on #242 and my latest updates I think we have everything covered.

**Pull request merged**